test/e2e/distributedtracing/distributedtracing_test.go

```diff
@@ -115,7 +117,18 @@ func TestServerSpanAndTraceIdInDecisionLog(t *testing.T) {
 			t.Fatalf("did not connect: %v", err)
 		}
 		client := ext_authz.NewAuthorizationClient(conn)
-		resp, err := client.Check(context.Background(), &req)
+		ctx := context.Background()
+
+		// mimicking how a grpc client would append the headers to the outgoing context
+		ctx = metadata.AppendToOutgoingContext(ctx, "x-b3-parentspanid", "2a2b3c4d5e6f7a8b")
+		ctx = metadata.AppendToOutgoingContext(ctx, "x-b3-traceid", exampleTraceID)
+		ctx = metadata.AppendToOutgoingContext(ctx, "x-b3-spanid", "3f6a0b6d9d5f4b45")
+		ctx = metadata.AppendToOutgoingContext(ctx, "x-b3-sampled", "1")
+		ctx = metadata.AppendToOutgoingContext(ctx, "X-B3-Flags", "1")
+		ctx = metadata.AppendToOutgoingContext(ctx, "X-B3-Baggage-User", "alice")
+		ctx = metadata.AppendToOutgoingContext(ctx, "X-B3-Baggage-Transaction", "12345")
```

Comment:
Do we have a reference from where these headers come from? It would helpful to add it as a comment.